pkgs/misc/themes/gtk2/gtk-engine-bluecurve/default.nix

```diff
@@ -0,0 +1,19 @@
+{ stdenv, fetchurl, pkgconfig, intltool, gtk2 }:
+
+stdenv.mkDerivation {
+  name = "gtk-engine-bluecurve-1.0";
+
+  src = fetchurl {
+    url = "http://ftp.gnome.org/pub/gnome/teams/art.gnome.org/archive/themes/gtk2/GTK2-Wonderland-Engine-1.0.tar.bz2";
+    sha256 = "1nim3lhmbs5mw1hh76d9258c1p923854x2j6i30gmny812c7qjnm";
+  };
+
+  buildInputs = [ pkgconfig intltool gtk2 ];
+
```

Comment:
As `pkgconfig` and `intltool` should only be needed at build time I believe they belong in `nativeBuildInputs`.